### PR TITLE
DC2 units playtest page (Robust)

### PR DESCRIPTION
**Pull request by IJW-Wartrader**

### CODE REVIEW

**Comment by digi-brain** [comment]
I thought I'd make this experimental/playtest content my first priority, with the intent of a 'light touch' as previously discussed.

However, I got sucked in and edited rather more than I'd first intended. Anyway, I believe that I have helped with the clarity and consistency -- hopefully I haven't upset things that you find frustrating.

File: modules/experimental/pages/robust.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+* When you calculate the detachment strength, count the unit as equal to its remaining Damage Capacity.
+* Each hit reduces its Damage Capacity by one point.
+* If the unit's Damage Capacity is reduced to 0 then it is destroyed.
+
+Where you attack a detachment that includes units that have the {robust} ability:
+
+* Each {robust} unit counts as *two* units for hit allocation.
+This means that when you allocate hit dice and you reach a {robust} unit, you must allocate *two* dice to that unit before you allocate dice to more distant targets.
+// TODO: Do we still have to allocate two dice even if the unit has only 1 DC remaining?
+* Where possible, you must allocate two dice that score equal to or higher than the {robust} unit's Armour value, *or* two dice that score lower.
```
Comment (IJW-Wartrader) [inline modules/experimental/pages/robust.adoc:34]: > TODO: Do we still have to allocate two dice even if the unit has only 1 DC remaining?

Hmm. I'm not sure. It probably makes sense to change it to the number of remaining DC, but so far it's come up very rarely.

```suggestion
This means that when you allocate hit dice and you reach a {robust} unit, you must allocate as many dice to that unit as it has remaining Damage Capacity, before you allocate dice to more distant targets.
* Where possible, all dice allocated to the {robust} unit must score equal to or higher than its Armour value, *or* all dice allocated must score lower.
```

File: modules/experimental/pages/robust.adoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
++
+NOTE: The aim of this restriction is to force the attacker to destroy the unit or bypass it, and so to minimise the need to track units that have only one remaining point of Damage Capacity.
+
+* You may not allocate a third hit die to the unit until you have first allocated a hit die to all eligible units, in the usual way.
+// TODO: Shouldn't this be "a third and fourth hit die" (or "more dice") to be consistent? Or, what if the unit has only 1 DC remaining -- should it then be only "a second die" (or "another die")?
```
Comment (IJW-Wartrader) [inline modules/experimental/pages/robust.adoc:39]: Agreed.

```suggestion
* You may not allocate a further hit die to the unit until you have first allocated hit dice to all eligible units, in the usual way.
```

File: modules/experimental/pages/robust.adoc (hunk 1 above)
Comment (digi-brain) [reply modules/experimental/pages/robust.adoc:34]: Okay, allocate against current/remaining DC. That's my gut feeling too.
Your suggestion looks good, I'd just like to tweak it a bit — and we also need to integrate line #31, which isn't included in the selection for this conversation thread. So, I'll make a new suggestion on a new thread that includes that line.

Comment (digi-brain) [inline modules/experimental/pages/robust.adoc:34]: New suggestion, per earlier thread. Based on yours, but with some tweaks to wording. I also added a little explanation of the last bullet in parentheses at the end — but I'm not wedded to it if you're not keen on that addition.

```suggestion
* Each {robust} unit counts as either *one* or *two* units for hit allocation, according to the number of Damage Capacity points it has left.
This means that when you allocate hit dice and you reach a {robust} unit, you must allocate two dice if it is undamaged or one die if it is damaged.
Then continue to allocate dice to more distant targets in the usual way.
* Where possible, all dice that you allocate to a {robust} unit must score equal to or higher than its Armour value, *or* all dice that you allocate must score lower.
(To minimise the number of damaged units that you need to track in the game.)
```

Comment (digi-brain) [reply modules/experimental/pages/robust.adoc:34]: Oops, didn't need that parenthetical bit as we've covered it in the note in the next line, which is probably a better way to do it anyway :-D

```suggestion
* Each {robust} unit counts as either *one* or *two* units for hit allocation, according to the number of Damage Capacity points it has left.
This means that when you allocate hit dice and you reach a {robust} unit, you must allocate two dice if it is undamaged or one die if it is damaged.
Then continue to allocate dice to more distant targets in the usual way.
* Where possible, all dice that you allocate to a {robust} unit must score equal to or higher than its Armour value, *or* all dice that you allocate must score lower.
```

Comment (IJW-Wartrader) [reply modules/experimental/pages/robust.adoc:34]: Looking good.

**Review by digi-brain — APPROVED** [review]
Good to go I reckon. I'll merge it in a moment.

**Pull request merged**